src/main/scala/is/hail/utils/HashMethods.scala

```diff
@@ -32,3 +32,45 @@ class TwoIndepHash32(outBits: Int, a: Long, b: Long) extends (Int => Int) {
 
   override def apply(key: Int): Int = ((a * key + b) >>> shift).toInt
 }
+
+object FiveIndepTabulationHash32 {
+  def apply(rand: RandomDataGenerator): FiveIndepTabulationHash32 =
+    new FiveIndepTabulationHash32(
+      Array.fill[Long](256 * 4)(rand.getRandomGenerator.nextLong()),
+      Array.fill[Int](261 * 3)(rand.getRandomGenerator.nextInt())
```

Comment:
did you want to change these to 259 vs 261?

Reply:
Yes, and good catch! I ran a loop to double check the math, that derivedKeyTable is never indexed into past 259. And as a quick speed check, it wasn't until I increased the loop to 200000000 that I noticed any slowdown in the scratch worksheet.